package.json

```diff
@@ -24,9 +24,10 @@
   },
   "private": true,
   "dependencies": {
-    "vue": "^1.0.21",
+    "babel-runtime": "^6.0.0",
```

Comment:
Let's all chat about that - I have it as another task. I've been putting it off so we can get the latest versions of libs, and then pin when we're getting closer to release, but I'm open to discussion on timing.